frontend/javascripts/oxalis/model/reducers/annotation_reducer.ts

```diff
@@ -260,19 +266,36 @@ function AnnotationReducer(state: OxalisState, action: Action): OxalisState {
       const meshInfo: MeshInformation = {
         segmentId: segmentId,
         seedPosition,
-        seedAdditionalCoordinates,
+        seedAdditionalCoordinates: seedAdditionalCoordinates,
         isLoading: false,
         isVisible: true,
         isPrecomputed: false,
         mappingName,
         mappingType,
       };
-      const addCoordString = getAdditionalCoordinatesAsString(state.flycam.additionalCoordinates);
-      const updatedKey = update(state, {
+      const additionalCoordinates = state.flycam.additionalCoordinates;
+      const additionalCoordKey = getAdditionalCoordinatesAsString(additionalCoordinates);
+
+      let stateWithCurrentAddCoords = state;
+
+      // maybe add current add. coord. as key for meshes in state
+      if (getMeshesForAdditionalCoordinates(state, additionalCoordinates, layerName) == null) {
+        stateWithCurrentAddCoords = update(state, {
+          localSegmentationData: {
+            [layerName]: {
+              meshes: {
+                [additionalCoordKey]: { $set: [] },
+              },
+            },
+          },
+        });
+      }
```

Comment:
wouldn't the same code be needed for the `ADD_PRECOMPUTED_MESH` case? if so, I'd move this into a helper function and call it from both sites.